macOS/DuckDuckGo/VisualRefresh/ColorPalette.swift

```diff
@@ -127,7 +59,7 @@ final class NewColorPalette: ColorPalette {
     let controlsDecorationPrimary: NSColor = .controlsDecorationPrimary
     let controlsDecorationSecondary: NSColor = .controlsDecorationSecondary
     let controlsDecorationTertiary: NSColor = .controlsDecorationTertiary
-    let highlightDecoration: NSColor = .highlightDecoration
+    let highlightPrimary: NSColor = .highlightDecoration
```

Comment:
In order to (easily) match the Figma exports, we’re renaming the following properties

- `toneTint` > `toneTintPrimary`
- `toneShade` > `toneShadePrimary`
- `accentGlow` > `accentGlowPrimary`
- `accentAltGlow` > `accentAltGlowPrimary`
- `highlightDecoration` > `highlightPrimary`